semantic_conventions/trace/tls.yaml

```
@@ -0,0 +1,204 @@
+groups:
+  - id: tls
+    prefix: tls
+    type: span
+    brief: "This document defines semantic conventions for TLS/SSL client and server Spans."
+    attributes:
+      - id: version
+        requirement_level: required
+        brief: >
+          Numeric part of the version parsed from the original string of the
+          negotiated [SSL/TLS protocol version](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_version.html#RETURN-VALUES)
+        type: string
+        examples: ["1.2", "3"]
+      - id: version_protocol
+        requirement_level: required
```

it'd be nice to have a default value (TLS) so that we don't need to report it for the most common case.  

```suggestion
        requirement_level:
          conditionally_required: if not `TLS`
```